examples/skysync-cli-example-create-job-01.ps1

```diff
@@ -0,0 +1,77 @@
+<#
+.DESCRIPTION
+    Sample script in PowerShell to showcase the use of the CLI to generate and manage SkySync jobs. Example scenario:
+        * Create an NFS connection
+        * Create a Box connection
+        * Create a SkySync job that utilizes these connections
+        * Start the job
+
+.REFERENCE
+    https://skysync.atlassian.net/wiki/spaces/SSP/pages/15302683/SkySync+Command-line+Interface
+
+.NOTES
+    Setup/Config:
+        1. Create file 'skysync-cli.json' and place one directory above cli project/script that is being executed and add the following JSON block to the file:
+
+            {
+	        "output-json": true,
+	        "server": "http://localhost:9090/",
+	        "username": "***********",
+	        "password": "***********"
+            } 
+        2. Configure connection auth files
+        3. Update source and destination paths
+                Pathing Usage:
```

Comment:
since this sample is for nfs to box, why is there a my computer example here?  also, can these example formats more closely correspond to the paths below?

Reply:
removed the My Comp example and updated paths to match